### PR TITLE
[ZH] Update `Guides\Getting_Your_Map_Modded`

### PR DESCRIPTION
**Pull request by YumeMuzi**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Guides/Getting_Your_Map_Modded/zh.md [hunk 1]
```diff
@@ -1,55 +1,73 @@
...
+私聊其他玩家也是找摸的一种好方法。通常，如果摸图员没有开设摸图队列帖，那么可以去他们的个人主页查阅有关摸图的信息。阅读完毕后，即可私聊他们发出请求。虽然这些摸图员可能很难联系到，但他们或许有其他方法来提供摸图，例如架设一个私人摸图平台等等。
 
-## 华语圈找 mod 其它途径
+所有 BN 都在他们的个人主页上标注了摸图信息。不管他们有没有架设摸图队列帖，私聊他们是请求摸图，甚至请求提名谱面的重要方式。
 
-### Pending吧
+如果不在意摸的质量，可能最简单的找摸方法是：请求你的朋友来摸图。询问拥有作图经验的朋友，你大概率能收到满意的结果。
 
-传送门：[Link](https://tieba.baidu.com/f?kw=pending)
+## 华语圈找摸
```
Comment (kj415j45) [inline wiki/Guides/Getting_Your_Map_Modded/zh.md:45]: Note for maintainer: this section contains many Simplified Chinese community specific content.

File: wiki/Guides/Getting_Your_Map_Modded/zh.md [hunk 2]
```diff
@@ -1,55 +1,73 @@
----
-outdated: true
----
+# 如何为你的谱面找摸
```
Comment (kj415j45) [inline wiki/Guides/Getting_Your_Map_Modded/zh.md:1]:
```suggestion
# 如何为你的谱面找摸（Mod）
```

File: wiki/Guides/Getting_Your_Map_Modded/zh.md [hunk 3]
```diff
@@ -1,55 +1,73 @@
...
+## #modreqs 频道
 
-## 互相 mod (M4M)
+*警告：不要在频道里刷屏请求，否则你有可能被[禁言](/wiki/Silence)！建议在 `#modreqs` 频道的发言频率低于每小时一次。*
 
-跟“等价交换”差不多，用自己一个 mod 换一个别人的 mod。通常这个是最有效的找 mod 方式。甚至在自己在 \#modreq 发完之后等另外一个人发的时候马上就有一个潜在的 modder 在这里。互相 mod 的同时也在互相进步，这何乐而不为？
+`#modreqs` 频道位于 osu! 聊天区。玩家能在这里通过发送 `/np` 指令快捷加链接，并附上摸图请求的种类，从而为自己的谱面找摸。即使这种方法效率不高 摸图员或者 BN 也会抽空浏览这个频道，挑选一些心仪的谱面。
 
-## 直接找目标 modder
+`#modreqs` 频道已不再有以前的功能和用户量。但即便如此，这个频道也是找摸的好途径，值得一试。虽然频道里的摸多为单向摸，但也能见到[上面](#types-of-mods)所提到的其他请求类型。
```
Comment (kj415j45) [inline wiki/Guides/Getting_Your_Map_Modded/zh.md:35]:
```suggestion
`#modreqs` 频道已不再有以前的功能和用户量。但即便如此，这个频道也是找摸的好途径，值得一试。虽然频道里的摸多为单向摸，但也能见到[上面](#摸图请求的种类)所提到的其他请求类型。
```
小节链接需要与译文一致

File: wiki/Guides/Getting_Your_Map_Modded/zh.md (hunk 1 above)
Comment (Walavouchey) [reply wiki/Guides/Getting_Your_Map_Modded/zh.md:45]: having community-specific info would in theory be fine, but directly linking to external groups could be questionable. this needs a few more opinions.

Comment (YumeMuzi) [reply wiki/Guides/Getting_Your_Map_Modded/zh.md:45]: deleted these links and keep the rest of description

**Review by kj415j45 — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**